README.md

```diff
@@ -1,5 +1,21 @@
-This repository serves as the collection-point, ideation hub, and process behind the [InnerSource Commons'](http://innersourcecommons.org) InnerSource Patterns--a set of proven and reviewed solutions to InnerSource problems. These patterns illustrate beneficial activities and behaviors found in [organizations who apply InnerSource methodologies](http://innersourcecommons.org/getting-started/). See below the list for more on what a pattern is, how to use them in your organization, and how to get involved.
+This repository contains the InnerSource Patterns collected by [innersourcecommons.org][isc-website]. These patterns illustrate beneficial activities and behaviors found in [organizations who apply InnerSource methodologies][isc-getting-started]. 
```

Comment:
```suggestion
This repository contains the InnerSource Patterns collected by the [innersourcecommons.org](InnerSource Commons). These patterns illustrate beneficial activities and behaviors found in [organizations who apply InnerSource methodologies][isc-getting-started]. 
```

Reply:
```suggestion
This repository contains the InnerSource Patterns collected by the [innersourcecommons.org](InnerSource Commons). These patterns illustrate beneficial activities and behaviors found in [organizations who apply InnerSource][isc-getting-started]
```
(This is @MaineC 's suggestion - minus the word methodologies)

Reply:
You suggestion won't create a valid link I think. But I understand what you are after and will fix it accordingly.

Reply:
Please see the latest version of this branch/PR.